test/mixin-spec.js

```diff
@@ -44,17 +44,13 @@ describe('Model', function() {
 
   });
 
-  describe('inheritable static methods', function() {
+  describe('prototype', function() {
```

Comment:
should be static method
`Model.getNextId = function(){...}`

not a prototype method `Model.prototype.getNextId = function(){...}`

or in es6
```
class Model {

  static getNextId(){
    ...
  }

}
```